services/repository/memory/service.go

```diff
@@ -3,52 +3,50 @@ package memory
 
 import (
 	"context"
+	"sort"
 
 	"github.com/decentralized-cloud/tenant/models"
 	"github.com/decentralized-cloud/tenant/services/repository"
 	"github.com/lucsky/cuid"
+	"github.com/micro-business/go-core/common"
+	"github.com/thoas/go-funk"
 )
 
-var tenants map[string]models.Tenant
-
 type repositoryService struct {
-}
-
-func init() {
-	tenants = make(map[string]models.Tenant)
+	tenants map[string]models.Tenant
```

Comment:
Why did the declaration move to Init?

Reply:
The declaration moved out of init, not into init due to the private, but global variable tenants in the package cause problem during unit tests, we discussed this last night during code walk through